docs/source/user_guide/user_guide/getting_started.md

```diff
@@ -0,0 +1,232 @@
+# Getting started
+
+# Installation
+
+## Pre-requisites
+- one of the operating systems described in [Supported systems](./supported_systems.md)
+- a supported Python version installed, and in the PATH
+- a supported GPU
+
+## Procedure
+```
+pip install gstaichi
+```
+## Sanity checking the installation
+```
+python -c 'import gstaichi as ti; ti.init(arch=ti.gpu)'
+```
+(should not show any error messages)
+
+# A first GsTaichi kernel
+
+Let's use a linear congruential generator - a pseudo-random number generator - since they are not easily possible for a compiler to optimize out. First, in normal python:
+
+```
+def lcg_np(B: int, lcg_its: int, a: npt.NDArray) -> None:
+    for i in range(B):
+        x = a[i]
+        for j in range(lcg_its):
+            x = (1664525 * x + 1013904223) % 2147483647
+        a[i] = x
+```
+We are taking in a numpy array, of size B, looping over it. For each value in the array, wwe run 1000 iterations of LCG, then update the original value.
+
+Let's write out the full code, including creating a numpy array, and timing this method:
+
+```
+import numpy as np
+import numpy.typing as npt
+import time
+
+
+def lcg_np(B: int, lcg_its: int, a: npt.NDArray) -> None:
+    for i in range(B):
+        x = a[i]
+        for j in range(lcg_its):
+            x = (1664525 * x + 1013904223) % 2147483647
+        a[i] = x
+
+B = 16000
+a = np.ndarray((B,), np.int32)
+
+start = time.time()
+lcg_np(B, 1000, a)
+end = time.time()
+print("elapsed", end - start)
+```
+
+You can find the full code also at [lcg_python.py](../../../../python/gstaichi/examples/lcg_python.py)
+
+On a Macbook Air M4, this gives the following output:
+```
+# elapsed 5.552601099014282 on macbook air m4
+```
+
+Now let's convert it to gstaichi
+
+Here is the function, written as a GsTaichi kernel:
```

Comment:
A `GsTaichi` kernel? Are you sure the `Gs` prefix is needed here? But as you wish.